### PR TITLE
CEDS-2238 - statistical_value page to GDS

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: app/assets/stylesheets/customsdecexfrontend-app.scss [hunk 1]
```diff
@@ -14,7 +14,10 @@
...
 
+
 $govuk-assets-path: '/customs-declare-exports/assets/';
 $govuk-images-path: "/customs-declare-exports/assets/lib/govuk-frontend/govuk/assets/images/";
 $govuk-fonts-path: "/customs-declare-exports/assets/lib/govuk-frontend/govuk/assets/fonts/";
-@import 'lib/govuk-frontend/govuk/all';
\ No newline at end of file
+@import 'lib/govuk-frontend/govuk/all';
+
+@import 'partials/_currency_input.scss';
```
Comment (MaciejRewera) [inline app/assets/stylesheets/customsdecexfrontend-app.scss]: Is it required to be imported after `govuk/all` ? If not, then it mightt be better to keep it before. GDS had some rules to be the last imported part.

File: app/assets/stylesheets/partials/_currency_input.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+span.prefix {
+  position: absolute;
+  padding: 8px;
+  margin-left: 2px;
+}
+input.input-prefix {
+  padding-left: 30px;
+}
```
Comment (PatrykRudnicki) [inline app/assets/stylesheets/partials/_currency_input.scss]: 👍 for new file for the styles 🎉

File: app/assets/stylesheets/partials/_currency_input.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
+span.prefix {
```
Comment (bambuchaAdm) [inline app/assets/stylesheets/partials/_currency_input.scss]: please put this into separate scope like `.input-currency` - would not apply accidentally

**Comment by SteveSugden** [comment]
I'm going to check with Chris about over-riding the govuk styles.

File: app/assets/stylesheets/partials/_currency_input.scss (hunk 3 above)
Comment (SteveSugden) [reply app/assets/stylesheets/partials/_currency_input.scss]: You mean change `span.prefix` to `span.input-currency`?   In fact, on reflection, I don't know why I didn't make the style names the same.   Any they aren't really limited to currency's - could be any prefix.

I'' have another look at the class names.

File: app/assets/stylesheets/customsdecexfrontend-app.scss (hunk 1 above)
Comment (SteveSugden) [reply app/assets/stylesheets/customsdecexfrontend-app.scss]: Moved it to above govuk.  Turns out it didn't need to be last to work.

File: app/assets/stylesheets/partials/_currency_input.scss (hunk 3 above)
Comment (SteveSugden) [reply app/assets/stylesheets/partials/_currency_input.scss]: Renamed the new css classes to be more specific to the role they are being used for.

File: app/assets/stylesheets/partials/_currency_input.scss [hunk 4]
```diff
@@ -0,0 +1,9 @@
+
+span.currency-prefix {
```
Comment (bambuchaAdm) [inline app/assets/stylesheets/partials/_currency_input.scss]: in scss you could write this as 
```
.currency-prefix {
  span {
    // code
  }
  input {
    // code
  }
}
```

Comment (SteveSugden) [reply app/assets/stylesheets/partials/_currency_input.scss]: Doesn't seem to work for some reason.  I'll leave as-is if that's ok

**Pull request merged**